rclpy/rclpy/action/client.py

```diff
@@ -589,13 +583,6 @@ def wait_for_server(self, timeout_sec=None):
 
     def destroy(self):
         """Destroy the underlying action client handle."""
-        if self._client_handle is None:
-            return
         with self._node.handle:
```

Comment:
No need for `with self._node.handle` here either

Reply:
Fixed in 7882366